scripts/ci/ci_build_dockerhub.sh

```diff
@@ -67,6 +67,25 @@ echo
 # shellcheck source=scripts/ci/_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/_script_init.sh"
 
+DOCKER_IMAGE_SPEC="${DOCKERHUB_USER}/${DOCKERHUB_REPO}:${DOCKER_TAG}"
+
+IMAGE_CREATED_SINCE=$(docker image ls "${DOCKER_IMAGE_SPEC}" --format '{{ .CreatedSince }}')
+
+if [[ ${IMAGE_CREATED_SINCE} =~ .*days.* || ${IMAGE_CREATED_SINCE} =~ .*weeks.* ||
+      ${IMAGE_CREATED_SINCE} =~ .*months.* || ${IMAGE_CREATED_SINCE} =~ .*years.* ||
+      ${IMAGE_CREATED_SINCE} =~ [0-9][0-9]\ hours ]]; then
```

Comment:
Does it show `09 hours` in the `docker image ls ${DOCKER_IMAGE_SPEC}" --format '{{ .CreatedSince }}'`?

Reply:
Nope. It's 9 hours :)

Reply:
Then 48 hours and above that there are days.

Reply:
```bash
❯ export tta="9 hours"
❯ if [[ "$tta" =~ [0-9][0-9]\ hours ]]; then
then> echo "Rebuilding the image."
then> else
else> echo "Skipping rebuilding the image."
else> fi
Skipping rebuilding the image.
```

Reply:
But that's OK. I wanted to rebuild it when we hit 10 ..

Reply:
9 hours is correct -> skipping....

Reply:
So basically I want to rebuild for everything = or above 10 hours.

Reply:
I am not sure how the 10 hours are calculated - i.e. I think 9h + 1 minute might be displayed as 10 hours

Reply:
I wanted to make aproper calculation from UTC time but doing it in bash is kinda complex